app/features-json/view_models/repo_view_model.rb

```diff
@@ -0,0 +1,23 @@
+require_relative "../../shared/json_convertible"
+
+module FastlaneCI
+  # View model to expose the basic info about repositories.
+  class RepoViewModel
+    include FastlaneCI::JSONConvertible
+
+    # @return [String]
+    attr_reader :full_name
+
+    # @return [String]
+    attr_reader :api_path
```

Comment:
Do we need the api path yet? After looking at the info required for the add project dialog, i'm thinking that we just need the url and full name for now.